src/main/java/crimson/CommitStatusHandler.java

```diff
@@ -0,0 +1,60 @@
+package crimson;
+import io.github.cdimascio.dotenv.Dotenv;
+import kotlin.Pair;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.client.HttpClient;
+import java.io.IOException;
+import java.util.*;
+
+public class CommitStatusHandler {
+    static Dotenv dotenv = Dotenv.load();
+
+    public static final String ERROR = "error";
+    public static final String FAILURE = "failure";
+    public static final String PENDING = "pending";
+    public static final String SUCCESS = "success";
+
+    private static final String USER_NAME = "641bill";
+    static final String GITHUB_TOKEN = dotenv.get("GITHUB_TOKEN");
```

Comment:
Let's make this to be injected from constructor.

I will inject it in main (and read from env in main)